buildstockbatch/postprocessing.py

```diff
@@ -595,6 +595,7 @@ def remove_intermediate_files(fs, results_dir, keep_individual_timeseries=False)
 def upload_results(aws_conf, output_dir, results_dir):
     logger.info("Uploading the parquet files to s3")
 
+    buildstock_dir = Path(results_dir).parent.joinpath('housing_characteristics')
```

Comment:
I think this is the problem right here. It's assuming that file is in the usual place it is on Eagle. I think the sampler returns the buildstock.csv location. We should catch that and use that file. Could be a bit tricky because the sample is called way before the postprocessing on a different node and everything.